project/Build.scala

```diff
@@ -304,16 +305,26 @@ object Finagle extends Build {
     libraryDependencies ++= jacksonLibs
   ).dependsOn(finagleCore, finagleHttp)
 
+  lazy val finagleZipkinCore = Project(
+    id = "finagle-zipkin-core",
+    base = file("finagle-zipkin-core"),
+    settings = Defaults.coreDefaultSettings ++
+      sharedSettings
+  ).settings(
+    name := "finagle-zipkin-core",
+    libraryDependencies ++= Seq(util("codec"), util("events")) ++ scroogeLibs,
```

Comment:
the pants build file also has a dep on util-app and util-core. util-core should def be added here. if this doesn't have a direct util-app dependency, then can you remove from the pants build file? (or vice-versa add it here).